### PR TITLE
Create a Json file which contain current pokemon, trainer, battle and…

### PR DESCRIPTION
**Pull request by adhes14**
… town

### CODE REVIEW

**Review by lechemon4000 — DISMISSED** [review]
Good job!!

**Review by adrianovi — APPROVED** [review]
Great!

**Review by Agustin-Mediotti — APPROVED** [review]
Nice 👍🏼

**Review by eliosfg — DISMISSED** [review]

**Review by eliosfg — APPROVED** [review]

File: src/main/java/org/fundacionjala/at15/pokemon/io/Reader.java [hunk 1]
```diff
@@ -80,6 +84,8 @@ public static File getPath(String fileName) {
                 return new File(BATTLE.getPath() + "/" + fileName + ".json");
             case "twn":
                 return new File(TOWN.getPath() + "/" + fileName + ".json");
+            case "crt":
+                return new File(TOWN.getPath() + "/" + fileName + ".json");
```
Comment (adrianovi) [inline src/main/java/org/fundacionjala/at15/pokemon/io/Reader.java:88]: Is it okay that is TOWN?

**Review by adrianovi — CHANGES_REQUESTED** [review]

**Review by adrianovi — APPROVED** [review]

**Review by jimytz — APPROVED** [review]
Well done!

**Pull request merged**